public/css/admin-parts-checkout.css

```diff
@@ -129,6 +131,7 @@ label {
 	display: flex;
 	align-items: center;
 	justify-content: center;
+	color: var(--green);
```

Comment:
Same thing about text color here. This is being inherited by the quantity `input`, so make sure to override it to black for the `input`